test/letters/containers/LetterList.unit.spec.jsx

```diff
@@ -11,27 +12,45 @@ const defaultProps = {
       letterType: 'commissary'
     },
     {
-      name: 'Benefit Summary Letter',
+      name: 'Benefit Summary and Service Verification Letter',
       letterType: 'benefit_summary'
     },
     {
       name: 'Benefit Verification Letter',
       letterType: 'benefit_verification'
     }
   ],
-  letterDownloadStatus: {}
+  lettersAvailability: AVAILABILITY_STATUSES.available,
+  letterDownloadStatus: {},
+  optionsAvailable: true,
 };
 
 describe('<LetterList>', () => {
-  it('should render', () => {
+  it('renders', () => {
```

Comment:
The pattern we've been using is `it('should...')`. Why remove the 'should'?

Reply:
Yeah, that's fair. I'll stick to the pattern going forward. Some of the test names got a little long-winded when written out this way (i.e. too much horizontal scrolling in my split panes) so I cut them all for consistency. Likely not necessary.